### ISSUE TITLE
Option to have background worker stop server after inactivity

### ISSUE THREAD

**Issue by Raekye** [Gamocosm/Gamocosm#10]
Prerequisites: flexible ping/query - there are many different version of Minecraft. Perhaps an emailer when server is supposed to be online but cannot be reached

**Comment by MarMed** [comment]
Any update?

**Comment by Raekye** [comment]
Got postponed dealing with some family stuff, started looking at it again sparsely over the past couple weeks, but can't give ETA :/ (ongoing things at home)
